### DISCUSSION TITLE
HDR in AR mode brightens the colors

### DISCUSSION THREAD

**Discussion by DenisAzarkin** [google/model-viewer#3716] [Q&A]
### Description

HDR in AR mode brightens the colors. For example: Blue cube in PC browser and AR. We have different color. On a screenshots was used HDR Neutral.

![image_2022-08-18_08-44-12](https://user-images.githubusercontent.com/23003266/185307703-19316632-4ec5-417e-ac27-5b8a88e870b4.png) ![photo_2022-08-18_08-58-16](https://user-images.githubusercontent.com/23003266/185305593-f52048f0-29df-4450-8037-9ed709cc259e.jpg)

### Version

- model-viewer: v1.12.0

### Browser Affected

- [ ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
You're using Scene Viewer; if you want consistent lighting, you'll need WebXR (which is our default mode). If you didn't intentionally change it, check your JS console for info. Usually the problem is not serving on HTTPS. 

Scene Viewer, Quick Look, and optionally WebXR, use estimated environment lighting to make the object look correct in its current surroundings, so consistency is explicitly not the goal; rather realism is. However, the ML lighting estimation has its own quirks, so it's not always better.
